sass/layout/_navigation.scss

```diff
@@ -328,6 +328,14 @@ a {
 		display: block;
 	}
 
+	a {
+		color: $color__link-navigation;
+
+		&:hover {
+			color: darken($color__link-navigation, 20%);
+		}
```

Comment:
I'm not a huge fan of declaring your colors this way. What if in a child theme we wanted to make the color complementary on hover, or something completely different? If we're going the route of including the parent theme styles in the child theme, with the aim of overriding things within the child theme's variables file then I think it's important to use a variable here like `$color__link-navigation-hover`.